SchemaDatabase/SGr/Generic/BaseTypes.xsd

```diff
@@ -454,199 +454,199 @@ https://github.com/SmartgridReady/SGrSpecifications  -->
+      <!-- <enumeration value="MEGAJOULES_PER_SQUARE_FOOT" /> -->
+      <!-- <enumeration value="MEGAJOULES_PER_SQUARE_METER" /> -->
+      <!-- <enumeration value="MEGAVOLT_AMPERES" /> -->
+      <!-- <enumeration value="MEGAVOLT_AMPERES_REACTIVE" /> -->
+      <!-- <enumeration value="MEGAVOLTS" /> -->
+      <!-- <enumeration value="MEGAWATT_HOURS" /> -->
+      <!-- <enumeration value="MEGAWATTS" /> -->
+      <!-- <enumeration value="MEGOHMS" /> -->
       <enumeration value="METERS" />
-      <enumeration value="METERS_PER_HOUR" />
-      <enumeration value="METERS_PER_MINUTE" />
+      <!-- <enumeration value="METERS_PER_HOUR" /> -->
+      <!-- <enumeration value="METERS_PER_MINUTE" /> -->
       <enumeration value="METERS_PER_SECOND" />
       <enumeration value="METERS_PER_SECOND_PER_SECOND" />
-      <enumeration value="MILES_PER_HOUR" />
-      <enumeration value="MILLIAMPERES" />
-      <enumeration value="MILLIBARS" />
-      <enumeration value="MILLIMETERS" />
-      <enumeration value="MILLIMETERS_OF_MERCURY" />
-      <enumeration value="MILLIMETERS_PER_MINUTE" />
-      <enumeration value="MILLIMETERS_PER_SECOND" />
-      <enumeration value="MILLIOHMS" />
-      <enumeration value="MILLISECONDS" />
-      <enumeration value="MILLIVOLTS" />
-      <enumeration value="MILLIWATTS" />
+      <!-- <enumeration value="MILES_PER_HOUR" /> -->
+      <!-- <enumeration value="MILLIAMPERES" /> -->
+      <!-- <enumeration value="MILLIBARS" /> -->
+      <!-- <enumeration value="MILLIMETERS" /> -->
+      <!-- <enumeration value="MILLIMETERS_OF_MERCURY" /> -->
+      <!-- <enumeration value="MILLIMETERS_PER_MINUTE" /> -->
+      <!-- <enumeration value="MILLIMETERS_PER_SECOND" /> -->
+      <!-- <enumeration value="MILLIOHMS" /> -->
+      <!-- <enumeration value="MILLISECONDS" /> -->
+      <!-- <enumeration value="MILLIVOLTS" /> -->
+      <!-- <enumeration value="MILLIWATTS" /> -->
       <enumeration value="MINUTES" />
-      <enumeration value="MONTHS" />
-      <enumeration value="NEWTON" />
-      <enumeration value="NEWTON_METERS" />
-      <enumeration value="NEWTON_SECONDS" />
-      <enumeration value="NEWTONS_PER_METER" />
+      <!-- <enumeration value="MONTHS" /> -->
+      <!-- <enumeration value="NEWTON" /> -->
+      <!-- <enumeration value="NEWTON_METERS" /> -->
+      <!-- <enumeration value="NEWTON_SECONDS" /> -->
+      <!-- <enumeration value="NEWTONS_PER_METER" /> -->
       <enumeration value="NO_UNITS" />
-      <enumeration value="OHM_METERS" />
+      <!-- <enumeration value="OHM_METERS" /> -->
       <enumeration value="OHMS" />
-      <enumeration value="PARTS_PER_BILLION" />
+      <!-- <enumeration value="PARTS_PER_BILLION" /> -->
       <enumeration value="PARTS_PER_MILLION" />
       <enumeration value="PASCALS" />
-      <enumeration value="PER_HOUR" />
-      <enumeration value="PER_MINUTE" />
-      <enumeration value="PER_SECOND" />
+      <!-- <enumeration value="PER_HOUR" /> -->
```

Comment:
`PER_HOUR` wird im `SontexSupercal5V0.2.1.xml` benutzt, Ändern? In was?

Reply:
wir können PER_HOUR auskommentieren